src/shared/Git-Credential-Manager/Git-Credential-Manager.csproj

```diff
@@ -4,7 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OSPlatform)'=='windows'">net461;netcoreapp3.1</TargetFrameworks>
-    <RuntimeIdentifiers>win-x86;osx-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win-x86;osx-x64;linux-x64</RuntimeIdentifiers>
```

Comment:
Git for Windows also works on x86 Windows, so we need to have a GCM Core that supports that RID.
At the same time, I didn't have headspace to go about updating the build/release/signing pipelines to support _two_ Windows channels. Since GCM Core isn't memory intensive we can live with x86.

_Also_ Windows on ARM can only translate x86 to ARM, not x86_64/AMD64 "x64", so targeting x86 on Windows is the most compatible.

If we can simplify the build/release/sign pipelines then we might want to do "the right thing" and publish for all available RIDs.